components/dashboard/src/settings/Integrations.tsx

```diff
@@ -547,8 +547,17 @@ export function GitIntegrationModal(props: ({
 
     const updateHostValue = (host: string) => {
         if (mode === "new") {
-            setHost(host);
-            setRedirectURL(callbackUrl(host));
+
+            let verifiedHost = host;
+
+            if (host.includes("https://")) {
+                verifiedHost = host.replace("https://","");
+            } else if (host.includes("http://")) {
```

Comment:
```suggestion
            } else if (host.startsWith("http://")) {
```

or use a regexp

Reply:
Yeah @JanKoehnlein i agree `startsWith` is a better method in this case. Would just make the changes. 

Github provides a UI to directly commit the suggestions, but I fear I might need two commits for the two suggestions. So i'd instead manually make the changes in one commit.